caravel/templates/caravel/dashboard.html

```diff
@@ -2,7 +2,8 @@
 
 {% block head_js %}
   {{ super() }}
-  <script src="/static/assets/dist/dashboard.entry.js"></script>
+  {% set VERSION_STRING = appbuilder.get_app.config.get("VERSION_STRING") %}
```

Comment:
it would be nice to DRY this up and make a helper method like `include_entry_javascript("dashboard")` that can be used in all our templates.